compiler/rustc_ast/src/attr/mod.rs

```diff
@@ -458,6 +459,16 @@ impl MetaItemKind {
             AttrArgs::Eq(_, AttrArgsEq::Hir(lit)) => Some(MetaItemKind::NameValue(lit.clone())),
         }
     }
+
+    pub fn defines(&self) -> Vec<NodeId> {
+        match self {
+            MetaItemKind::Word => vec![],
+            MetaItemKind::List(things) => {
+                things.iter().filter_map(|i| Some(i.meta_item()?.path.segments[0].id)).collect()
+            }
+            MetaItemKind::NameValue(_) => vec![],
+        }
+    }
```

Comment:
Is this function ever used?
The ids used to identify `defines` paths are created with `next_node_id` in resolver, so they do not match ids kept in meta-item paths (if such ids are filled to non-dummy values at all).